packages/tour-kit/src/components/tour-kit-step.tsx

```diff
@@ -29,6 +34,19 @@ const TourKitStep: React.FunctionComponent< Props > = ( {
 		config.steps[ currentStepIndex ].options?.className
 	);
 
+	useEffect( () => {
+		if ( typeof config.options?.callbacks?.onStepViewOnce !== 'function' ) {
+			return;
+		}
```

Comment:
Nit - I think it might be better to move this to the end and let the rest of the logic work irrespective. Keeping track of viewed steps is something that could happen regardless of callback.

So could be:

```
typeof config.options?.callbacks?.onStepViewOnce === 'function' && config.options?.callbacks?.onStepViewOnce( currentStepIndex );
```

Thinking whether to actually even avoid the "function" check and let it error if misused in this case. 😬

Reply:
Cool, I did not think that keeping track of viewed steps was something we'd need if no callback function was provided, but it's something that could be useful 👍